All EIPIP Meetings/Meeting 011.md

```diff
@@ -0,0 +1,552 @@
+
+**Pooja Ranjan**
+So if we have to look into the post mortem, we just gave a look to the poll that we did earlier that was in a form of project. But in one of the previous EIPIP meetings, we decided that this kind of information should be made public in the form of an EIP. We can save it as an informational EIP, which is way easier accessible for people rather than finding it in some particular GitHub repository in some projects section. So this is a draft that we have created for the previous Muir Glacier upgrade, and we have tried to provide as much information as possible starting from the upgrade summary section. Like what date, time, block number, and who owns that particular block.  The details from there included a list of the EIPs that were included, but for Muir Glacier, it was just the only one for pushing up the difficulty bomb. So we use that. And this is the sequence of events that took place, the decision making date so that people can go and refer. The thought is to make everything accessible for the community later on to read. And if we come up with some good practices that we would want to continue in the future, some corrective actions like the rubric that we discussed, about how we can prevent anything mess happening or what was the issues we faced and what about that possible suggestions to that...
+
+**James Hancock**
+Are you in a position you could share The screen with the document that we're talking about?
+
+**Pooja Ranjan**
+I think I can do that. Is it visible now?
+
+**James Hancock**
+Yeah, I can see it. Okay.
+
+**Pooja Ranjan**
+I also shared the link in the chat for people to refer. So we have not submitted it. It's just a draft, and it is staying with us. I wanted to discuss this with you guys to collect some feedback if anything is missing. The two questions I have in mind, I'll just mention it so that if you're giving it a look, or maybe think about it, the discussion section here, what we did is we refer to the eth-magician post that James created for Muir Glaicer upgrade. We did not create any specific eth-magician post to just list the Muir Glacier, because I'm not sure if there is much more discussion needed or if we can just combine both of them. So that was one. And the other thing was at the end of this document, we have also listed the resources. This is not a typical format of an EIP. Do we need to have it? If it is fine to keep it, because I think I have mentioned all the links here already, then we will keep it. Otherwise, we might want to remove it. And yes, please feel free to share your thoughts about this. There's one more thing. We have this template. This is the general template for future post mortem upgrades. This would also be posted in the form of an informational EIP for people to refer and when we are creating, we'll keep this in mind. I will share the link of this document as well. So feel free to add any comment. Anyone have any questions?
```

Comment:
I also shared the [link in the chat](I also shared the link in the chat for people to refer to. So we have not submitted it. It's just a draft, and it is staying with us. I wanted to discuss this with you guys to collect some feedback if anything is missing. The two questions I have in mind, I'll just mention it so that if you're giving it a look, or maybe think about it, the discussion section here, what we did is we refer to the eth-magician post that James created for Muir Glacier upgrade. We did not create any specific eth-magician post to just list the Muir Glacier, because I'm not sure if there is much more discussion needed or if we can just combine both of them. So that was one. And the other thing was at the end of this document, we have also listed the resources. This is not a typical format of an EIP. Do we need to have it? If it is fine to keep it, because I think I have mentioned all the links here already, then we will keep it. Otherwise, we might want to remove it. And yes, please feel free to share your thoughts about this. There's one more thing. We have [this template](https://docs.google.com/document/d/1onC1rUqXTI3MNYJRiwhgLS91OaYrVgAlZb0Ri3i4hRE/edit). This is the general template for future upgrades postmortem. This would also be posted in the form of an informational EIP for people to refer and when we are creating, we'll keep this in mind. I will share the link to this document as well. So feel free to add any comments. Anyone have any questions?) for people to refer to. So we have not submitted it. It's just a draft, and it is staying with us. I wanted to discuss this with you guys to collect some feedback if anything is missing. The two questions I have in mind, I'll just mention it so that if you're giving it a look, or maybe think about it, the discussion section here, what we did is we refer to the eth-magician post that James created for Muir Glaicer upgrade. We did not create any specific eth-magician post to just list the Muir Glacier, because I'm not sure if there is much more discussion needed or if we can just combine both of them. So that was one. And the other thing was at the end of this document, we have also listed the resources. This is not a typical format of an EIP. Do we need to have it? If it is fine to keep it, because I think I have mentioned all the links here already, then we will keep it. Otherwise, we might want to remove it. And yes, please feel free to share your thoughts about this. There's one more thing. We have this template. This is the general template for future post mortem upgrades. This would also be posted in the form of an informational EIP for people to refer and when we are creating, we'll keep this in mind. I will share the link to this document as well. So feel free to add any comments. Does anyone have any questions?

https://github.com/poojaranjan/Ethereum-EIPs/blob/master/EIPS/Muir%20Glacier%20upgrade%20postmortem.md

Reply:
You added link markup, but we don't know what link to use for this, at the top.

Reply:
https://docs.google.com/document/d/1onC1rUqXTI3MNYJRiwhgLS91OaYrVgAlZb0Ri3i4hRE/edit

Added the link in the later part of the conversation "We have this template."